src/components/Lesson/Lesson.js

```diff
@@ -15,6 +18,16 @@ const Lesson = React.createClass({
       __html: this.props.lesson.content
     };
   },
+  componentWillMount(){
```

Comment:
This function minifies to:

```
            componentWillMount: function() {
                if (!t) {
                    var e = this.props.lesson.content;
                    a(e) && (this.props.lesson.content = o(e))
                }
            },
```

Setting a breakpoint at `if (!t)` yields the value of `t` from the closure:

![skjermbilde 2016-07-31 kl 14 56 50](https://cloud.githubusercontent.com/assets/4810521/17276643/3df274d8-572f-11e6-9764-2a2a5fe7755d.png)

Any ideas? Some webpack wizardry?